src/DssSpell.sol

```diff
@@ -19,26 +19,14 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-import { MCD } from "dss-test/MCD.sol";
-import { D3MInit, D3MCommonConfig, D3MAaveConfig } from "src/dependencies/dss-direct-deposit/D3MInit.sol";
-import { D3MInstance } from "src/dependencies/dss-direct-deposit/D3MInstance.sol";
-
-interface VestLike {
-    function restrict(uint256) external;
-    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
-}
-
-interface GemLike {
-    function allowance(address, address) external view returns (uint256);
-    function approve(address, uint256) external returns (bool);
-}
+import "dss-interfaces/dapp/DSTokenAbstract.sol";
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/54731a9e15d6da097d578d8050bec900cf7223f5/governance/votes/Executive%20Vote%20-%20February%2022%2C%202023.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/3ccd2e061217ef336e1cf1d71b9cfcce36548f74/governance/votes/Executive%20vote%20-%20March%208%2C%202023.md -q -O - 2>/dev/null)"
```

Comment:
I will reiterate that wasn't the original point of the commit hash. It was only to ensure the content is available for reading.
But honestly to avoid keep wasting time in this irrelevant discussion I modified it to the merge commit.